addOns/fuzz/src/main/java/org/zaproxy/zap/extension/fuzz/httpfuzzer/HttpFuzzerHandler.java

```diff
@@ -176,7 +181,7 @@ private HttpFuzzer createFuzzer(
                 processors);
     }
 
-    private String createFuzzerName(HttpMessage message) {
+    protected String createFuzzerName(HttpMessage message) {
```

Comment:
Do i need to Overrider this method ? or it is ok to have same fuzzer name ?

Reply:
It's fine to have the default name, but if you think it would be better to show a custom one we can allow to specify one when showing the dialogue.

Reply:
I think then it is ok to have same name for now.